vignettes/bubble_tab_bar/lib/navbar_button.dart

```diff
@@ -14,15 +14,15 @@ class NavbarButton extends StatefulWidget {
   final bool isSelected;
   final VoidCallback onTap;
 
-  const NavbarButton(this.data, this.isSelected, {@required this.onTap });
+  const NavbarButton(this.data, this.isSelected, {required this.onTap });
 
   @override
   _NavbarButtonState createState() => _NavbarButtonState();
 }
 
 class _NavbarButtonState extends State<NavbarButton> with SingleTickerProviderStateMixin {
-  AnimationController _iconAnimController;
-  bool _wasSelected;
+  late AnimationController _iconAnimController;
+  bool _wasSelected = false;
```

Comment:
I think this should probably be:
```
late bool _wasSelected = widget.isSelected;
```
Not sure how much it matters, but I think is technically more correct.

Reply:
I'll just land this now and fix locally.

Reply:
Yes it is probably more appropriate, thanks